backend/api/Helpers/Exceptions/BadRequestException.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace Pims.Api.Helpers.Exceptions
+{
+    /// <summary>
+    /// BadRequestException class, provides a way to handle bad request exceptions so that they are returned by the middleware in a standardized way.
+    /// </summary>
+    public class BadRequestException : Exception
```

Comment:
Throwing this is better than returning a `BadRequest` because our middleware will wrap the message in a standardized JSON error.